src/main/java/com/esri/samples/na/find_service_areas_for_multiple_facilities/FindServiceAreasForMultipleFacilitiesSample.java

```diff
@@ -0,0 +1,224 @@
+    ArcGISFeatureTable facilitiesTable = new ServiceFeatureTable("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0");
+    // create a feature layer from the table
+    FeatureLayer facilitiesFeatureLayer = new FeatureLayer(facilitiesTable);
+
+    // create a symbol used to display the facilities
+    PictureMarkerSymbol facilitySymbol = new PictureMarkerSymbol("http://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png");
+    facilitySymbol.setHeight(50);
+    facilitySymbol.setWidth(50);
+    // set the renderer of the facilities feature layer to use the facilities symbol
+    facilitiesFeatureLayer.setRenderer(new SimpleRenderer(facilitySymbol));
+
+    // add the facilities feature layer to the map
+    map.getOperationalLayers().add(facilitiesFeatureLayer);
+
+    // wait for the facilities feature layer to load
+    facilitiesFeatureLayer.addDoneLoadingListener(() -> {
+      if (facilitiesFeatureLayer.getLoadStatus() == LoadStatus.LOADED) {
+
+        // zoom to the extent of the feature layer
+        mapView.setViewpointGeometryAsync(facilitiesFeatureLayer.getFullExtent(), 130);
+
+        // wait for the view to zoom to enable the ui
+        mapView.addDrawStatusChangedListener(new DrawStatusChangedListener() {
+          @Override
+          public void drawStatusChanged(DrawStatusChangedEvent drawStatusChangedEvent) {
+            if (drawStatusChangedEvent.getDrawStatus() == DrawStatus.COMPLETED) {
+              // enable the 'find service areas' button
+              findServiceAreasButton.setDisable(false);
+              mapView.removeDrawStatusChangedListener(this);
+            }
+          }
+        });
+
+        // resolve 'find service areas' button click
+        findServiceAreasButton.setOnAction(event -> {
+
+          // disable the button
+          findServiceAreasButton.setDisable(true);
+
+          // show the progress indicator
+          progressIndicator.setVisible(true);
+
+          // create a service area task from URL
+          ServiceAreaTask serviceAreaTask = new ServiceAreaTask("https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ServiceArea");
+          serviceAreaTask.loadAsync();
+
+          // create default service area task parameters
+          ListenableFuture<ServiceAreaParameters> serviceAreaTaskParametersFuture = serviceAreaTask.createDefaultParametersAsync();
+          serviceAreaTaskParametersFuture.addDoneListener(() -> {
+            try {
+              ServiceAreaParameters serviceAreaParameters = serviceAreaTaskParametersFuture.get();
+              // set the task parameters to have the task return polygons
+              serviceAreaParameters.setPolygonDetail(ServiceAreaPolygonDetail.HIGH);
+              serviceAreaParameters.setReturnPolygons(true);
+              // add a service area of 3 minutes in addition to the default service area of 5 minutes
```

Comment:
I find the wording here a little confusing. How about something like:
"add a service area of 3 minutes travel time by car in addition to the default of 5 minutes travel time"